src/Contact.scss

```diff
@@ -17,11 +17,12 @@
     @media (min-width: 901px) {
         display: flex;
         flex-direction: row; 
```

Comment:
if we say display:flex 
by default 
        flex-direction: row; 

so, you don't have to write it